src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -741,12 +747,24 @@ def _add_manifest_entry(self,
             )
         })
 
-    def _parse_file_id_column(self, file_id: Optional[str]) -> Optional[str]:
+    def _parse_drs_uri(self,
+                       file_id: Optional[str],
+                       descriptor: JSON
+                       ) -> Optional[str]:
+        try:
```

Comment:
If TDR somehow provides a `file_id` we should honor and use that. We should only fall back to the external DRS URI if TDR doesn't provide one. This wasn't spec'ed anywhere.